### DISCUSSION TITLE
drop-down menus

### DISCUSSION THREAD

**Discussion by YongShen-98** [luizdepra/hugo-coder#813] [Q&A]
Hello, does your theme support drop-down menus? If I want to do it, where do I change it? Thanks

**Comment by luizdepra** [comment]
Sadly, not yet.

Add a multilevel menu is not so easy, but you could try to use [this discussion](https://discourse.gohugo.io/t/how-to-create-multilevel-dropdown-menu-in-hugo/18373) as a base to start/

**Reply by YongShen-98** [reply]
Thanks for u reply! I just did it on the final .html
